zerver/lib/event_queue.py

```diff
@@ -198,15 +214,19 @@ def cleanup(self):
 descriptors_by_handler_id = {} # type: Dict[int, ClientDescriptor]
 
 def get_descriptor_by_handler_id(handler_id):
+    # type: (int) -> ClientDescriptor
     return descriptors_by_handler_id.get(handler_id)
 
 def set_descriptor_by_handler_id(handler_id, client_descriptor):
+    # type: (int, ClientDescriptor) -> None
     descriptors_by_handler_id[handler_id] = client_descriptor
 
 def clear_descriptor_by_handler_id(handler_id, client_descriptor):
+    # type: (int, ClientDescriptor) -> None
```

Comment:
Here `client_descriptor` can be `None`. Therefore, `client_descriptor` should be `Optional[ClientDescriptor]`.